repo/packages/K/kafka/800/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "2.6.0-2.2.1"
```

Comment:
I wonder, does an upgrade from `2.5.0-2.1.0` also work? That's outside the scope of this PR but would be good to know as it could make it easier on our customers to upgrade.

Reply:
The upgrade from 2.5.0-2.2.1 -> 2.6.0-2.2.1 -> 2.7.0-2.3.0 works. Not sure if we can upgrade directly from 2.5.0-2.2.1.

Reply:
It has to be upgraded from `2.5.0-2.1.0` -> `2.6.0-2.2.1` first then to `2.7.0-2.3.0`

Reply:
I think as long as the protocol version is the same it would work. Let's continue this conversation out of this PR